mapper/pojo-standalone/src/main/java/org/hibernate/search/mapper/pojo/standalone/schema/management/impl/SearchSchemaManagerImpl.java

```diff
@@ -56,6 +58,16 @@ public void dropAndCreate() {
 		doOperation( PojoScopeSchemaManager::dropAndCreate );
 	}
 
+	@Override
+	public void exportExpectedSchema(SearchSchemaCollector collector) {
+		delegate.exportExpectedSchema( collector );
+	}
+
+	@Override
+	public void exportExpectedSchema(Path targetDirectory) {
+		delegate.exportExpectedSchema( targetDirectory );
+	}
```

Comment:
It's odd, Sonar reports this lines as not covered in tests: https://sonarcloud.io/component_measures?metric=new_coverage&selected=org.hibernate.search%3Ahibernate-search-parent%3Amapper%2Fpojo-standalone%2Fsrc%2Fmain%2Fjava%2Forg%2Fhibernate%2Fsearch%2Fmapper%2Fpojo%2Fstandalone%2Fschema%2Fmanagement%2Fimpl%2FSearchSchemaManagerImpl.java&view=list&pullRequest=3437&id=org.hibernate.search%3Ahibernate-search-parent

but from what I can see they should be, because of `integrationtest/mapper/pojo-standalone-realbackend/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/standalone/realbackend/schema/management/ElasticsearchSchemaManagerExporterIT.java` in particular.

Could you check those tests are actually getting executed?

Maybe it's just a problem of this integration test module (which is relatively new, I believe) not being included in the aggregated test coverage report... no dependency for that module in `build/reports/pom.xml`?

Reply:
tests are being executed:
```
[INFO] --- maven-failsafe-plugin:3.0.0-M8:integration-test (it-elasticsearch) @ hibernate-search-integrationtest-mapper-pojo-standalone-realbackend ---
[INFO] Using configured provider org.apache.maven.surefire.junitcore.JUnitCoreProvider
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.hibernate.search.integrationtest.mapper.pojo.standalone.realbackend.EntityAsGraphSmokeIT
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.672 s - in org.hibernate.search.integrationtest.mapper.pojo.standalone.realbackend.EntityAsGraphSmokeIT
[INFO] Running org.hibernate.search.integrationtest.mapper.pojo.standalone.realbackend.EntityAsTreeSmokeIT
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.537 s - in org.hibernate.search.integrationtest.mapper.pojo.standalone.realbackend.EntityAsTreeSmokeIT
[INFO] Running org.hibernate.search.integrationtest.mapper.pojo.standalone.realbackend.mapping.DecimalScaleMappingIT
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.027 s - in org.hibernate.search.integrationtest.mapper.pojo.standalone.realbackend.mapping.DecimalScaleMappingIT
[INFO] Running org.hibernate.search.integrationtest.mapper.pojo.standalone.realbackend.schema.management.ElasticsearchSchemaManagerExporterIT
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.057 s - in org.hibernate.search.integrationtest.mapper.pojo.standalone.realbackend.schema.management.ElasticsearchSchemaManagerExporterIT
```
so hopefully the problem is indeed in the missing dependency in the reporting module

Reply:
Looks like this fixed it. Thanks!